crates/parser/src/ix_parser/vixen_ix/structure.rs

```diff
@@ -0,0 +1,323 @@
+use serde_json::Value;
```

Comment:
The transaction decompose mat be better spot for core or really where the similar logic for accounts resides.

Reply:
moved to core